### PR TITLE
Cat Project: Adds update project descriptions.

### PR DESCRIPTION
**Pull request by castavridis**
References a lot of the past instruction. Added some more Git detail to Part 1. Made Part 2 more mobile-first. Tried to remove overt references to desktop-first approach.

- When will mentees push their commits?
-- Should they be taught push their commits right away?

- Now references the updated project in my repository since I couldn't update the original: https://github.com/castavridis/web_group_project_1

### CODE REVIEW

**Review by ktmathews89 — APPROVED** [review]
I updated lesson 7 :) Thanks Christine! 

I removed all the Udacity responsive course stuff bc they already had a lesson on responsive development.

**Comment by castavridis** [comment]
@ktmathews89 Nice! I know that Udacity was removed but I'm wondering about the updates in this commit: https://github.com/ktmathews89/cg_summer_2018/pull/1/commits/b21607349e37ff22b3072b24e04a3ee627d0a178

File: lesson_archive/4_cat_costume_2.md [hunk 1]
```diff
@@ -40,34 +55,76 @@ First, add the below viewport meta element in your html's <head></head>. This is
 
 Next, add the Google Fonts you need!
 
-### Course Material for Challenge 3
+While you're coding your CSS, try to reuse as much CSS as possible. The wireframing step should help you identify common styles. Here is an example of consolidating repetitive CSS inspired by [CSS Wizardry](https://csswizardry.com/2013/07/writing-dryer-vanilla-css/):
```
Comment (castavridis) [inline lesson_archive/4_cat_costume_2.md]: Do we want to have reminder about consolidating CSS?

File: lesson_archive/4_cat_costume_2.md [hunk 2]
```diff
@@ -40,34 +55,76 @@ First, add the below viewport meta element in your html's <head></head>. This is
...
-#### Challenge 4: Use Media Queries to Help with "Breaks"
+#### Challenge 4: Use media queries to make the site responsive
 
 In today's world, we rarely make sites just adaptive. Usually we want the site to look good on ALL device and don't usually target just one particular device and screen size. This means we want our site to be RESPONSIVE.
 
-**STEP 1: Determine widths where site breaks down**
+**STEP 1: Make the site look good at sizes larger than 1200px.
 
-Start resizing your screen (by dragging in the window size) from big to small. When you notice something "break", like 2 boxes sliding under each other or elements overlapping, then make note of the window width. You can see width width in the top right hand corner of your browser window but ONLY when you have developer tools opened.
+It seems like the desktop site was designed for screen sizes of 1200px. Try to replicate the desktop design inside of a media query for widths larger than 1200px.
```
Comment (castavridis) [inline lesson_archive/4_cat_costume_2.md]: I updated these steps to be more mobile-first friendly since the older instructions were desktop-first.

**Pull request merged**